pages/index.mdx

```diff
@@ -1,6 +1,6 @@
 import { Steps,Callout } from 'nextra/components'
 
-# About ASoC
+# About alias summer of code
```

Comment:
Capatilization of ALiAS is incorrect!
Can you fix it?

Reply:
sure thing

Reply:
Looks good!